feature/policy_forwarding/otg_tests/ip_guev1_static_decap_subnet_range/README.md

```diff
@@ -0,0 +1,260 @@
+| Flow type#6 | IP+UDP+IPv6 Payload | Outer IPGUE    | ATE-P1-Address   | ATE-P2-Address                                                                             | Varies depending on the application     | 6081 (DUT configured decap port)         | 35   | 70  |
+|             |                       | Inner IPv6     | IPV6-SRC1        | IPV6-DST1                                                                                  | N/A                                     | N/A                                          | 32   | 50  |
+
+*  ATE Port 2 receives below flow/packet types:
+
+| RX-Flow-type   | Payload Type        | Header Level   | Source IP        | Destination IP   | DSCP | TTL | UDP Source Port                             | UDP Destination Port                             |
+|----------------|---------------------|----------------|------------------|------------------|------|-----|-----------------------------------------|----------------------------------------------|
+| RX-Flow-type#1 | IPv4 Payload        | N/A            | IPV4-SRC1        | IPV4-DST1        | 32   | 49  | N/A                                     | N/A                                          |
+| RX-Flow-type#2 | IPv6 Payload        | N/A            | IPV6-SRC1        | IPV6-DST1        | 32   | 49  | N/A                                     | N/A                                          |
+| RX-Flow-type#5 | IP+UDP+IPv4 Payload | Outer IPGUE    | ATE-P1-Address   | ATE-P2-Address   | 35   | 68  | Varies depending on the application     | 6081 (configured non-default on DUT)         |
+|                |                     | Inner IPv4     | IPV4-SRC1        | IPV4-DST1        | 32   | 50  | N/A                                     | N/A                                          |
+| RX-Flow-type#6 | IP+UDP+IPv6 Payload | Outer IPGUE    | ATE-P1-Address   | ATE-P2-Address   | 35   | 68  | Varies depending on the application     | 6081 (configured non-default on DUT)         |
+|                |                     | Inner IPv6     | IPV6-SRC1        | IPV6-DST1        | 32   | 50  | N/A                                     | N/A                                          |
+ 
+ ### DUT Configuration
+
+1.  Interfaces: Configure all DUT ports as singleton IP interfaces towards the ATE ports
+ 
+2. GUE Decapsulation:
+    *  Configure a Loopback address with DECAP-DST_3/32 address
+    *  Configure the DUT as GUEv1 decapsulator with below fields:
+       - DECAP-DST-SUBNET/30 as decapsulation subnet-range
+          - DECAP-DST/32 is one of the IP prefix from DECAP-DST-SUBNET/30
+       - Do not enable copying TTL bits from outer to inner header post decapsulation
+       - Use single UDP destination port 6081 for both IPv4 and IPv6 payload type
+    *  Configure static routes for destination IPV4-DST2 and IPV6-DST2 towards ATE Port 2
+    *  Configure the UDP decap destination port as 6081 
```

Comment:
This is redundant to line 86 above https://github.com/openconfig/featureprofiles/pull/4178/files#diff-e5956eca55ce603152d1d2a717707b75ec9740dfe7edd4e430213d3de4085b24R86